app/services/samples/metadata/file_import_service.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'roo'
+
+module Samples
+  module Metadata
+    # Service used to import sample metadata via a file
+    class FileImportService < BaseService
+      SampleMetadataFileImportError = Class.new(StandardError)
+
+      def initialize(project, user = nil, params = {})
+        super(user, params)
+        @project = project
+        @file = params[:file]
+        @sample_id_column = params[:sample_id_column]
+        @ignore_empty_values = params[:ignore_empty_values] || false
```

Comment:
We shouldn't need to explicitly set false here. If it doesn't find the param it will be set to nil which we can use

Reply:
0455419f8ab52f8a4ce8956eda4a29c21195e97b